lib/core/widget/cases_search_widget.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:dental_ui/core/utils/constant/icon_constant.dart';
+
+class CasesSearchWidget extends StatelessWidget {
+  final void Function()? onSearchPressed;
+
+  const CasesSearchWidget({
+    Key? key,
+    this.onSearchPressed,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 115,
+      child: TextField(
+        decoration: InputDecoration(
+          labelText: 'Cases',
```

Comment:
hide text property

Reply:
used hint text, so once user starts typing the hint disappear.